.github/workflows/run_cell-type-wilms-tumor-14.yml

```diff
@@ -33,41 +33,22 @@ jobs:
   run-module:
     if: github.repository_owner == 'AlexsLemonade'
     runs-on: ubuntu-latest
+    container: public.ecr.aws/openscpca/cell-type-wilms-tumor-14:latest
 
     steps:
-      - name: Checkout repo
-        uses: actions/checkout@v4
-
-      - name: Set up R
-        uses: r-lib/actions/setup-r@v2
-        with:
-          r-version: 4.4.0
-          use-public-rspm: true
-
-      - name: Set up pandoc
-        uses: r-lib/actions/setup-pandoc@v2
-
-      - name: Install system dependencies
+      - name: Install git
         run: |
-          sudo apt-get install -y \
-            jags \
-            libcurl4-openssl-dev \
-            libfribidi-dev \
-            libglpk40 \
-            libharfbuzz-dev \
-            libhdf5-dev \
-            libmagick++-dev \
-            libtiff5-dev
+          apt-get update
+          apt-get install -y git
```

Comment:
What is git used for in this module, and why not add it to Dockerfile directly? I don't see anything particular in the scripts directory that needs the executable, so mostly I am just curious what's up here.

Reply:
git is used to check out the repository. If it is not there, then checkout step uses the API, which is less efficient. It might have even failed once, but I'm not sure I remember exactly.

Reply:
> If it is not there, then checkout step uses the API, which is less efficient. 

Ahhhh this makes sense. As the repo continues to grow, maybe we consider adding this to more workflows then

Reply:
Looked at the failure: yes, it failed because when using the API for checkout, no `.git` directory is created, which means that we can't use that to find the root directory.

Reply:
oh that certainly _does_ make sense!